src/Tribe/Tickets.php

```diff
@@ -3959,7 +3959,8 @@ public function maybe_redirect_to_attendees_registration_screen( $redirect = nul
 			$url = $attendee_reg->get_url();
 
 			if ( ! empty( $q_provider ) ) {
-				$url = add_query_arg( 'provider', $q_provider, $url );
+				$provider_slug = Tribe__Tickets__Tickets::get_provider_url_slug();
```

Comment:
```suggestion
				$provider_slug = tribe_tickets_get_provider_query_slug();
```
We don't have that method within the Ticket Class anymore, so that will return an error.